test/math_test.dart

```diff
@@ -79,6 +83,22 @@ void main() {
       expect(simResult, equals(true));
     }
 
+    test('power', () async {
+      await runMathVectors([
+        Vector({'a': 1, 'b': 100}, {'a_power_b': 1}),
+        Vector({'a': 1043, 'b': 0}, {'a_power_b': 1}),
+        Vector({'a': 3, 'b': 3}, {'a_power_b': 27}),
+        Vector({'a': 2, 'b': 1}, {'a_power_b': 2}),
+        Vector({'a': 2, 'b': 7}, {'a_power_b': 128}),
+        Vector({'a': 2}, {'a_power_const': 32}),
+        Vector({'a': 5, 'b': 3}, {'a_power_b': 125}),
+        Vector({'a': 7, 'b': 2}, {'a_power_b': 49}),
+        Vector({'a': 0, 'b': 0}, {'a_power_b': 1}),
+        Vector({'a': 0, 'b': 3}, {'a_power_b': 0}),
+        Vector({'a': 1, 'b': 10000000}, {'a_power_b': 1}),
```

Comment:
The width of `b` is 8 bits, so this value is getting truncated down.  It doesn't really matter a lot since the base is 1, but just in case you were trying to test large numbers on it here.  I feel like it's not a big deal to change since we're covering the math part of it in `LogicValue` tests already, will leave to your judgement.